required/latex-lab/latex-lab-math.dtx

```diff
@@ -198,15 +198,15 @@
 %
 %    \car{Change description here?}
 %    \begin{macrocode}
-\ProvidesFile{latex-lab-mathtagging.ltx}
-             [\ltmathtaggingdate\space
-              \ltmathtaggingversion\space
+\ProvidesFile{latex-lab-math.ltx}
+             [\ltmathdate\space
+              \ltmathversion\space
```

Comment:
in some sense this shows that we went the wrong way around ie going from XXX-tagging to just XXX. We should have probably done it the other way around, but I don't think we want to change that now. However, I think these 2 commands should be changed back to ltmathtagging... as ltXXXdate is plain wrong and might conflict with ltXXXdate used elsewhere (e.g. there is an ltmath.dtx file even though it currently doesn't  use the commands.

Reply:
What about say `\ltlabmathdate`  everywhere? That would never clash. I'm not so keen about "tagging" as while the tagging project triggers the changes, the code will have an impact on untagged documents too.

Reply:
fine